docs/app-dev/app-architecture.md

```diff
@@ -18,10 +18,9 @@ ABCI application, which is the logic that actually runs on the blockchain.
 Transactions sent by an end-user application are ultimately processed by the ABCI
 application after being committed by the Tendermint consensus.
 
-The end-user application in this diagram is the Cosmos Voyager, at the bottom
-left. Voyager communicates with a REST API exposed by a local Light-Client
-Daemon. The Light-Client Daemon is an application specific program that
-communicates with Tendermint nodes and verifies Tendermint light-client proofs
+The end-user application in this diagram is the [Lunie](https://lunie.io/), at the bottom
```

Comment:
But doesn't the image still show the cosmos voyager? - Also I don't think is the Lunie sounds right - shouldn't it be the Lunie App or something like that

Reply:
yes the image is still voyager, thought it might be better to link to something that is around instead of something that is not. That image is still used in tutorials, and we just say its lunie.

Reply:
```suggestion
The end-user application in this diagram is the [Lunie](https://lunie.io/) app, at the bottom
```

Reply:
lol okay